airbyte-integrations/connectors/destination-vectara/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "airbyte-destination-vectara"
+version = "0.2.2"
+description = "Airbyte destination implementation for Vectara"
+authors = ["Airbyte <contact@airbyte.io>"]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/destinations/vectara"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+
+[[tool.poetry.packages]]
+include = "destination_vectara"
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = ">=0.62.0"
```

Comment:
I think we should pin this to the latest-tested version.

Any version less than 1.0 should be considered unstable, per semver rules: https://semver.org/#spec-item-4

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

I have no problem pinning to `>=1.0,<2.0` after CDK has a 1.0 release, but I don't want to be chasing failures when breakages inevitably creep in.

```suggestion
# Note: Maintainers can bump this to the latest via `poetry add airbyte-cdk@latest`
airbyte-cdk = "0.74.0"
```

Long-run, another good option is dependabot-like process, and in that case, each bump would be automated, while still getting CI tests rerun before the version is bumped.